connectors/connector-google/server.ts

```diff
@@ -1,36 +1,27 @@
-// import type {HubspotSDK} from '@opensdks/sdk-google'
+import {initGoogleSDK} from '@opensdks/sdk-google'
 import type {ConnectorServer} from '@openint/cdk'
 import type {googleSchemas} from './def'
 
 export const googleServer = {
-  // newInstance: ({settings, fetchLinks}) => {
-  //   const sdk = initHubspotSDK({
-  //     // We rely on nango to refresh the access token...
-  //     headers: {
-  //       authorization: `Bearer ${settings.oauth.credentials.access_token}`,
-  //     },
-  //     links: (defaultLinks) => [
-  //       (req, next) => {
-  //         if (sdk.clientOptions.baseUrl) {
-  //           req.headers.set(
-  //             nangoProxyLink.kBaseUrlOverride,
-  //             sdk.clientOptions.baseUrl,
-  //           )
-  //         }
-  //         return next(req)
-  //       },
-  //       ...fetchLinks,
-  //       ...defaultLinks,
-  //     ],
-  //   })
-  //   return sdk
-  // },
-  // passthrough: (instance, input) =>
-  //   instance.request(input.method, input.path, {
-  //     headers: input.headers as Record<string, string>,
-  //     params: {query: input.query},
-  //     body: JSON.stringify(input.body),
-  //   }),
+  newInstance: ({settings}) => {
+    const sdk = initGoogleSDK({
+      headers: {
+        authorization: `Bearer ${settings.oauth.credentials.access_token}`,
+      },
+    })
+    return sdk?.drive_v3
+  },
+
+  async proxy(instance, req) {
+    return instance
+      .request(req.method as 'GET', req.url.replace(/.+\/api\/proxy/, ''), {
```

Comment:
Does the proxy work in your tests btw?

Reply:
haven't tested it as i got blocked with the upstream auth issues, will circle back here